definitions/modules.yaml

```diff
@@ -0,0 +1,36 @@
+# Modules for interpreted languages.
+
+name: PythonModules
+doc: Python module files.
```

Comment:
The intent of this isn't obvious to me. Do you want just .py's? Do you just want information about eggs? Do you want .pyc's and .so's? Do you actually want to download everything, or just list the files?